system_test/helpers/aest_docker.erl

```diff
@@ -27,10 +27,11 @@
 %=== MACROS ====================================================================
 
 -define(CONFIG_FILE_TEMPLATE, "epoch.yaml.mustache").
--define(EPOCH_CONFIG_FILE, "/home/epoch/epoch.yaml").
```

Comment:
Testing of backward compatibility of sync is currently not tested, and is tracked as https://www.pivotaltracker.com/story/show/163220278